### ISSUE TITLE
Product creation on HU sandbox error

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#10033]
Hi there, 

we were testing product card creation on HU sandbox but we got back an error message "No product was found for the given language of the offer. Please try again later." even though we are sending "hu-HU" as a language. 

Trace ID of the error is: 270e038eafceaa9e

Thank you in advance for your help! 

All the best, 

Dominika

**Comment by MaciejFrackowiak** [comment]
Based on the data you gave us in the request, we found an existing product with id=556f2c74-2f78-479e-9335-1300f45e4e34. However, a product you created in March 2024 doesn't have a Hungarian translation, so we cannot list an offer with "hu-HU" language. In such case, you should list an offer with "cs-CZ" language, and then add manually Hungarian translation.

**Comment by dominikahr** [comment]
@MaciejFrackowiak but at first I got the error message saying that product with this ID was not found in the Allegro catalogue, that is why I tried to create the product card. Why did I get this error message then? 

Also for Hungary it does not work the same way as on the other markets that the product data will be translated automatically by Allegro?

**Comment by MaciejFrackowiak** [comment]
Can you give me a trace-id of your request, where we returned an error message? As I see now, GET /sale/products/{productId} returns details of the product with if=556f2c74-2f78-479e-9335-1300f45e4e34.
For Hungary, it should work in the same way as for other marketplaces but for some products, we cannot have the translations yet.

**Comment by dominikahr** [comment]
@MaciejFrackowiak I tried to upload the offer again I again got the error message that the product with this ID is not in the Allegro catalogue, the trace ID of the error is: 1949cfe94a8b54ee

**Comment by MaciejFrackowiak** [comment]
Our response was correct - we couldn't find a product with such an id with a Hungarian translation.

**Comment by dominikahr** [comment]
@MaciejFrackowiak but up until now this error message was understood from our side in that way, that we should create a product card for the given product. But you said that the correct way in this case is to " list an offer with "cs-CZ" language, and then add manually Hungarian translation". 

I do not understand how we should do this. When Hungarian seller wants to list this offer, we get the error message "Product with this ID was not found in the Allegro catalogue", when we give the seller the option to create product card, we get the error "No product was found for the given language of the offer. Please try again later." So in any case we are not able to publish the offer and the seller has the product data only in Hungarian and he wants to sell on HU storefront. What is the expected solution here please?

**Comment by MaciejFrackowiak** [comment]
The seller should wait until the Hungarian translation is added (sometime in the future) or contact Allegro support to ask to add it manually by the support request. You have to remember that in the production environment, such a case will appear rather than in the Sandbox.

**Comment by dominikahr** [comment]
@MaciejFrackowiak ok thank you :) And how will it work with the trasnaltions? You mentioned "For Hungary, it should work in the same way as for other marketplaces but for some products, we cannot have the translations yet." - so everything will be translated but it will take some time?

**Comment by MaciejFrackowiak** [comment]
Yes, indeed. As a final result, whole products from our product catalog should have translations in all the marketplaces languages.
